src/Transformers/JsonApi/resource/relationship/relationshipGetType.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param {{id: String, type: String, data: Object}} relation
+ *
+ * @returns {String|undefined}
+ */
+export default function relationshipGetType(relation) {
+  if (!relation.data || (Array.isArray(relation.data) && relation.data.length === 0)) {
+    return undefined;
```

Comment:
Ik zou hier null returnen om aan te geven dat wij de value expliciet op null zetten. 
Er is wat debat over maar ik geloof dat het best practice is om zelf nooit iets op undefined te zetten.